### DISCUSSION TITLE
Location Bug Github Student Pack

### DISCUSSION THREAD

**Discussion by jerome-rutinowski** [community/community#42246] [GitHub Education] [answered]
Hello everyone,

I am unable to apply for student status because the form always tells me that "To continue you must modify your browser settings to allow GitHub to use your location information."
However, my browsers (Firefox 106.0.5, Safari 15.3) do permit github access to my location. I have tried to apply on multiple devices (iOS, macOS) and cannot get past this error. If you could please help me, so that I can finally apply, I would be very grateful.

**Comment by xixilaladsds** [comment, marked as answer]
Maybe your browser can't access the system permission of location services. You need to set it in system > Security & Privacy.

**Reply by jerome-rutinowski** [reply]
Thank you so much, that was indeed the problem! Even though this is a very easy fix, I am going to leave this thread/question online, so that people might profit from it in the future :)

**Reply by samus-aran** [reply]
Thanks @xixilaladsds for answering and awesome stuff @jerome-rutinowski for keeping this thread up 👍 🎉

**Reply by apple-enjoyer667** [reply]
Yeaaaaah Apple's Safari do not have access to Apple's iPhone Built-in Localisation Services that makes a 1000000% sense

**Comment by tully-8888** [comment]
If others encounter the same problem, try to enable Location services from MacOS if you disabled them.

**Reply by Saqibadnan0011** [reply]
This works for me Thanks @tully-8888 😃

**Reply by KwisatzJim** [reply]
I'm getting the same error. macOS 14.1.1, Firefox 120. 
Location settings in macOS are set to allow Firefox to use Location. 
https://education.github.com has been set to allow location. 
I've restarted Firefox, refreshed the page, , cleared cache, but still get "To continue you must modify your browser settings to allow GitHub to use your location information."

**Reply by KwisatzJim** [reply]
got past that by doing it on my phone with cellular instead of wifi

**Comment by Endermanbugzjfc** [comment]
Facing this now: https://github.com/orgs/community/discussions/60438
Every step is properly set, still got the same error.

**Comment by mebasoglu** [comment]
Hello, after allowing location access I still have this issue. I enabled location services on my operating system, I allowed pop up to give location access to Github but I still get the error. 

I tested my geolocation api access with this web site and it works: https://browserleaks.com/geo

I tried with Firefox, Google Chrome on Ubuntu and Firefox Mobile and Google Chrome Mobile on Android. I allowed location access every time.

**Reply by mebasoglu** [reply]
I tried to apply on Windows 10 this time and it worked. I can't believe that you can't apply GitHub Student Pack on GNU/Linux. Such a shame GitHub.

**Comment by kuanjiahong** [comment]
After implementing the solution provided in this discussion, I still encountered the **Modify your browser settings** error message,  here is what I did that seem to work (for some reason)

Here is what I did that resolve the error

My Operating Systems: Windows 11
The browser I used: Firefox

Prerequisite:
1. Make sure you enable location access in your browser AND your system as mentioned in this discussion

Steps:
1. Use a scanner to scan my student id card into a `.jpeg` file. If you have a `.pdf` file instead, you can just take a screenshot of it and save it as a `.jpeg` file

3. Upload the student id card `.jpeg` file

If the above steps fail, then the next thing you could do is to go to your campus physically, and repeat the steps (at least this is what I did)


I have no idea why my solution is able to solve an error message that is related to location access. But it works. 
Hope this help for the people who have been stuck in their application because of this error message

**Comment by Qunut999** [comment]
Hello!

I applied many times to join global campus and my application got rejected because of location access but now when I enabled my location and trying to resubmit again it shows.

Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.




And my rejected application showing "Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again". but how can I cancel my application? there is no option to cancel the application.

**Comment by schichtnudelauflauf** [comment]
Same here. All settings are on. Other websites can see my location.

**Comment by hybp** [comment]
Not working on MacOS. All privacy, location settings are on. Tried on Safari, iPad, Chrome, all of them didn't work. I just get a message that location settings are not met.

**Comment by kerwino** [comment]
I've tried many times on macOS and can't figure it out. I opened a Windows 10 virtual machine and it was done quickly

**Comment by d3jv** [comment]
Didn't work on linux or android as well. Doing it in a windows vm is probably the way to go.

**Comment by Tagvi** [comment]
In my case (Zen browser (Firefox)), the browser had enabled Privacy & Security > Permissions > Location > Settings > Block new requests asking to access your location. Turning it off fixed the issue

**Comment by peachmontt** [comment]
Make sure that both Firefox and Safari have location services enabled for GitHub

**Comment by iterating** [comment]
Not working on Android Chrome or Edge. Unable to validate student account

**Comment by Geronimo-Basso** [comment]
Hey guys, I have the same problem and couldn't solved it, so I switch to my iPhone and did it there, works perfectly.

**Comment by MiniduOshan** [comment]
change the browser

**Comment by didvc** [comment]
So **only laptops would work** do you guys **mean??** hell, why just Sensors -> Location patching doesn't work? Google Maps works with it, geez.

**Comment by gdphoenix108** [comment]
I don't know if it's helpful but I had exactly the same issue with my Ipad not being able to share my location.
What I did is: Go to Chrome---> settings ----> privacy and security ----> Safe Browsing ----> Change the Standard protection to No protection. After you succesfully submitted you application you can go back and change the settings into Standard protection. Hope this helps :)

**Comment by idris129** [comment]
Hi, 

I face problem each time whem I aply can you help me.


thank you





![Uploading Screenshot_20260822_070022_com_google_android_gm_MailActivityGmail.jpg…]()

**Comment by adrinake** [comment]
Estou tendo o problema de alegar que meus dados nao estao corretos, mesmo o nomes e endereços estando correto, nao aceitam minhas fotos mostrando ser EAD e nem meu atestado de matricula, sempre da mensagem generica de verificar os dados de nomes e documentos
